charts/consul/test/unit/connect-inject-clusterrole.bats

```diff
@@ -51,6 +51,9 @@ load _helpers
   local actual=$(echo $object | yq -r '.resources[| index("namespaces")' | tee /dev/stderr)
   [ "${actual}" != null ]
 
+  local actual=$(echo $object | yq -r '.resources[| index("nodes")' | tee /dev/stderr)
```

Comment:
I had added 'node' back to the connect inject cluster role in a previous PR (yesterday) and I missed the bats test for it.